tests/common.py

```
@@ -359,6 +359,7 @@ def run(
 
     # run snakemake
     if shellcmd:
+        breakpoint()
```

Ruff (0.15.13)

[error] Trace found: `breakpoint` used

(T100)